scripts/bump-version.js

```diff
@@ -4,12 +4,12 @@
  * Updates the version of this package to the CLI specified version.
  */
 
-const { execSync } = require('child_process');
 const fs = require('fs');
 const path = require('path');
 
-const PACKAGE_JSON_FILE = path.resolve(__dirname, '..', 'package.json');
-const PACKAGE_LOCK_FILE = path.resolve(__dirname, '..', 'package-lock.json');
+const rootDir = path.resolve(__dirname, '..');
+const PACKAGE_JSON_FILE = path.resolve(rootDir, 'package.json');
+const PACKAGE_LOCK_FILE = path.resolve(rootDir, 'package-lock.json');
```

Comment:
Just one minor suggestion, these variables should be lowercased to maintain consistency with other scripts. What do you think?

```suggestion
const packageJsonFile = path.resolve(rootDir, 'package.json');
const packageLockFile = path.resolve(rootDir, 'package-lock.json');
```

Reply:
Good point, this PR seems suitable for such a change :slightly_smiling_face: